### DISCUSSION TITLE
TimePicker and DateTimePicker

### DISCUSSION THREAD

**Discussion by josch0** [mantinedev/mantine#2055] [Feature requests]
DatePicker is a great component to quickly and easily select a date or range by mouse click.

It would be great if there was a matching TimePicker where I can also select the time by mouse.

Usually you store date and time in a single variable in your model. Therefore it would be very useful if there would be a common DateTimePicker that would return date and time together as value.

There are dozens of good JavaScript DateTimePicker components on the internet, I would like to see a native Mantine alternative for this.

Something like this would be really cool! :-)
![image](https://user-images.githubusercontent.com/104630690/183618899-67d1943b-ce88-41d2-8dc2-ba6f754a3f7f.png)

**Comment by josch0** [comment]
Any response on this?
Thanks.

**Reply by rtivital** [reply]
It's a complex component, it is not planned to be released in 5.x, in 6.0 maybe since there are planned changes to dates package
